src/gossip.rs

```diff
@@ -0,0 +1,236 @@
+use std::sync::Arc;
+
+use anyhow::anyhow;
+use bitcoin::Network;
+use bitcoin_hashes::hex::{FromHex, ToHex};
+use lightning::util::ser::{ReadableArgs, Writeable};
+use log::{debug, info, warn};
+use reqwest::Client;
+use rexie::{ObjectStore, Rexie, TransactionMode};
+use wasm_bindgen::JsValue;
+
+use crate::error::{MutinyError, MutinyStorageError};
+use crate::logging::MutinyLogger;
+use crate::node::{NetworkGraph, RapidGossipSync};
+use crate::wallet::get_rgs_url;
+
+pub(crate) const GOSSIP_DATABASE_NAME: &str = "gossip";
+pub(crate) const GOSSIP_OBJECT_STORE_NAME: &str = "gossip_store";
+pub(crate) const GOSSIP_SYNC_TIME_KEY: &str = "last_sync_timestamp";
+pub(crate) const NETWORK_GRAPH_KEY: &str = "network_graph";
```

Comment:
How does this compare with `ldkstorage.rs` with the `NETWORK_KEY` value there?

Reply:
Ah yeah I guess I should delete that now because we aren't using LocalStorage for the network graph now